resources/assets/sass/components/_news.scss

```diff
@@ -0,0 +1,110 @@
+// News
+//
+// News blocks
+//
+// Markup:
+//<div class="c-news {$modifiers}">
+//    <a href="#" class="c-news__image-wrap">
+//        <img src="assets/img/news1.jpg" alt="" class="c-news__image">
+//    </a>
+//    <h3 class="c-news__title">
+//        <a href="#" class="c-news__title-link">Euroopa Kohus otsustas – lennuki tehniline rike ei päästa hüvitise maksmisest</a>
+//    </h3>
+//    <div class="c-news__meta">
+//        <p class="c-news__meta-date">Täna 12:31</p>
+//    </div>
+//</div>
+//
+// default   - A news block.
+// .m-small   - A small news block.
+//
+// Styleguide 6
+
+
+.c-news {
+
+	margin-bottom: 24px;
+
+	@include respond-to(tablet) {
+
+		margin-bottom: 48px;
+	}
+}
+
+	.c-news__image-wrap {
+
+		position: relative;
+		overflow: hidden;
+		display: block;
+		height: 192px;
+		margin-bottom: 24px;
+
+		.c-news.m-small & {
+
+			height: 144px;
+		}
+	}
+
+		.c-news__image {
+
+			position: absolute;
+			top: 50%;
+			left: 50%;
+			display: block;
+			max-width: 120%;
+			height: auto;
+			transform: translate(-50%, -50%);
+		}
+
+	.c-news__title {
+
+		font-size: $font-size-large;
+		font-weight: $font-weight-bold;
+		line-height: 1.15;
+		margin-bottom: 24px;
+
+		.c-news.m-small & {
+
+			font-size: $font-size-medium;
+		}
+	}
+
+		.c-news__title-link {
+
+			color: $color-text-link;
+			text-decoration: none;
+
+			&:hover {
+
+				text-decoration: underline;
+			}
+		}
+
+	.c-news__meta {
+
+		color: $color-text-neutral-dark;
+		font-size: $font-size-small;
+		position: relative;
+
+		&:after {
+
+			content: "";
```

Comment:
Properties should be ordered background, content, display, height, margin-top, position, top, width<br>Prefer single quoted strings